scene/gui/color_picker.cpp

```diff
@@ -152,11 +157,36 @@ void fragment() {
 
 	COLOR = vec4(mix(vec3(1.0), clamp(abs(fract(vec3((a - TAU) / TAU) + vec3(1.0, 2.0 / 3.0, 1.0 / 3.0)) * 6.0 - vec3(3.0)) - vec3(1.0), 0.0, 1.0), ((float(sqrt(x * x + y * y)) * 2.0)) / 1.0) * vec3(v), (b + b2 + b3 + b4) / 4.00);
 })");
+
+	circle_ok_color_shader.instantiate();
+	circle_ok_color_shader->set_code(OK_COLOR_SHADER + R"(
+// ColorPicker ok color hsv circle shader.
+
+uniform float v = 1.0;
+
+void fragment() {
+	float x = UV.x - 0.5;
+	float y = UV.y - 0.5;
+	x += 0.001;
+	y += 0.001;
+	float b = float(sqrt(x * x + y * y) < 0.5);
+	x -= 0.002;
+	float b2 = float(sqrt(x * x + y * y) < 0.5);
+	y -= 0.002;
+	float b3 = float(sqrt(x * x + y * y) < 0.5);
+	x += 0.002;
+	float b4 = float(sqrt(x * x + y * y) < 0.5);
+    float s = sqrt(x * x + y * y);
+    float h = atan(y, x) / (2.0*M_PI);
+    vec3 col = okhsl_to_srgb(vec3(h, s, v));
```

Comment:
```suggestion
	float s = sqrt(x * x + y * y);
	float h = atan(y, x) / (2.0*M_PI);
	vec3 col = okhsl_to_srgb(vec3(h, s, v));
```
Wrong indentation.

Reply:
I wonder why the lint tool didn't catch this.